pyproject.toml

```diff
@@ -8,7 +8,9 @@ author = "John Reese"
 author-email = "john@noswap.com"
 description-file = "README.rst"
 home-page = "https://aiosqlite.omnilib.dev"
-requires = []
+requires = [
+    "typing_extensions >= 4.0; python_version < '3.11'",
```

Comment:
And change this to `python_version < '3.8'` since 3.8 was the first version supporting `Literal`.